src/workers/userActivity.worker/inactivityTracker/index.js

```diff
@@ -1,61 +1,47 @@
-const { query: Hasura } = require("../../utils/hasura");
-const insertActivity = require("../../utils/insertUserActivity");
-const { getLastAppOpenedTimestamp } = require("./queries/queries");
+const { query: Hasura } = require('../../../utils/hasura');
+const insertActivity = require('../../../utils/insertUserActivity/');
+const { getLastAppOpenedTimestamp } = require('./queries/queries');
+const BATCH_SIZE = 500;
```

Comment:
Add BATCH_SIZE to config.js & use it from the ENV variable. Don't make it required, & set the default to 500